lib/config/generator.js

```diff
@@ -0,0 +1,210 @@
+var path = require('path');
+var Vow = require('vow');
+var vowNode = require('vow-node');
+var Table = require('cli-table');
+var prompt = require('prompt');
+var fs = require('fs');
```

Comment:
Nit-pick, could we change sequence of those definition like we do in other modules?

Like this:

```js

// Default node modules
var path = require('path');
var fs = require('fs');

// Deps
var Vow = require('vow');
var vowNode = require('vow-node');
var Table = require('cli-table');
var prompt = require('prompt');

// Our stuff
var Checker = require('../checker');
var utils = require('../utils');
```